TypeCobol.Test/Parser/Programs/Cobol85/CopyReplace4ColonMultiCopy.rdzPGM.txt

```diff
@@ -0,0 +1,20 @@
+﻿--- Program ---
+PROGRAM: CPYRPL4C common:False initial:False recursive:False
+ author: ? written: ? compiled: ? installation: ? security: ?
+--- Intrinsic:Namespace:Program:Global:Local
+-- DATA --------
+  xxxENT:Alphanumeric
+  xxxENT-FCT01:Alphanumeric
+  xxxENT-FCT01:Alphanumeric
+  xxxENT-FCT01-Var1:Alphanumeric
+  xxxENT-FCT01-Var1:Alphanumeric
+  xxxENT2:Alphanumeric
+  xxxENT2-FCT01:Alphanumeric
+  xxxENT2-FCT01-Var1:Alphanumeric
+  xxxENT1:Alphanumeric
```

Comment:
Not sure I understand how these variables are sorted but they are all there.

Reply:
Nothing to say